examples/shadowmap/main.js

```diff
@@ -1137,8 +1146,7 @@
             var shadowSettings;
             if ( this._config[ 'atlas' ] ) {
 
-                shadowSettings = this._shadowSettings[ num ];
-                shadowMap = this._shadowMapAtlas.addLight( light, shadowSettings );
+                shadowMap = this._shadowMapAtlas.addLight( light );
```

Comment:
Indeed !
Perhaps that check (https://github.com/cedricpinson/osgjs/blob/Kuranes/feature/ShadowAtlas_D3D-2581/sources/osgShadow/ShadowMapAtlas.js#L342) in shadowMap::addLight should do a warning rather than a return (light already added"